### PR TITLE
[Android] UIExplorer- Add tel and sms for IntentAndroid

### PR DESCRIPTION
**Pull request by offbye**

### CODE REVIEW

File: Examples/UIExplorer/IntentAndroidExample.android.js [hunk 1]
```diff
@@ -65,6 +65,8 @@ var IntentAndroidExample = React.createClass({
         <OpenURLButton url={'http://www.facebook.com'} />
         <OpenURLButton url={'http://facebook.com'} />
         <OpenURLButton url={'geo:37.484847,-122.148386'} />
+        <OpenURLButton url={'tel:10086'} />
+        <OpenURLButton url={'sms:10086'} />
```
Comment (mkonicek) [inline Examples/UIExplorer/IntentAndroidExample.android.js]: What's this number?

File: Examples/UIExplorer/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -3,6 +3,11 @@
           package="com.facebook.react.uiapp" >
 
   <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+  <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
```
Comment (mkonicek) [inline Examples/UIExplorer/android/app/src/main/AndroidManifest.xml]: Why are these three needed?

**Comment by mkonicek** [comment]
Thanks a lot for adding this!

**Comment by christopherdro** [comment]
@offbye Any chance you can update the PR based on the comments?

Thanks!

**Comment by satya164** [comment]
@offbye Any updates on this?

**Comment by satya164** [comment]
Closing since no activity on the PR. let's re-open if you wanna work on it again.

**Comment by leoabacade** [comment]
is it possible to send sms with body in it?

just like LinkingIOS?
```
var url = 'sms:'+contacts.phoneNumber+'&body=yes we can';
LinkingIOS.openURL(url);
```

**Comment by leoabacade** [comment]
ok, got it working

```
var url2 = 'sms:'+contacts.phoneNumber+'?body=yes we can';
IntentAndroid.openURL(url2);
```

**Pull request closed without merging**